tests/python/test_atari_env.py

```diff
@@ -311,7 +311,7 @@ def test_terminal_signal(tetris_env):
 
 
 def test_render_exception(tetris_env):
-    with pytest.raises(TypeError):
+    with pytest.raises(gymnasium.error.ResetNeeded):
```

Comment:
Could you add an `tetris_env.reset()` beforehand such that we have the same error